src/tests/ftest/util/mdtest_utils.py

```diff
@@ -0,0 +1,273 @@
+        #  -N=0                  stride num between neighbor tasks for file/dir
+        #                        operation (local=0)
+        #  -p=0                  pre-iteration delay (in seconds)
+        #  --random-seed=0       random seed for -R
+        #  -s=1                  stride between number of tasks for each test
+        #  -V=0                  verbosity value
+        #  -w=0                  bytes to write each file after it is created
+        #  -W=0                  number in seconds; stonewall timer, write as
+        #                        many seconds and ensure all processes did the
+        #                        same number of operations (currently only
+        #                        stops during create phase)
+        # -x=STRING              StoneWallingStatusFile; contains the number
+        #                        of iterations of the creation phase, can be
+        #                        used to split phases across runs
+        # -z=0                   depth of hierarchical directory structure
+
+        self.api = FormattedParameter("-a {}")
+        self.branching_factor = FormattedParameter("-b {}")
+        self.test_dir = FormattedParameter("-d {}")
+        self.barriers = FormattedParameter("-B {}")
+        self.read_bytes = FormattedParameter("-e {}")
+        self.first_num_tasks = FormattedParameter("-f {}")
+        self.iteration = FormattedParameter("-i {}")
+        self.items = FormattedParameter("-I {}")
+        self.last_num_tasks = FormattedParameter("-l {}")
+        self.num_of_files_dirs = FormattedParameter("-n {}")
+        self.pre_iter = FormattedParameter("-p {}")
+        self.random_seed = FormattedParameter("--random-seed {}")
+        self.stride = FormattedParameter("-s {}")
+        self.verbosity_value = FormattedParameter("-V {}")
+        self.write_bytes = FormattedParameter("-w {}")
+        self.stonewall_timer = FormattedParameter("-W {}")
+        self.stonewall_statusfile = FormattedParameter("-x {}")
+        self.depth = FormattedParameter("-z {}")
+
+        # Module DAOS
+        # Required arguments
+        #  --daos.pool=STRING            pool uuid
+        #  --daos.svcl=STRING            pool SVCL
+        #  --daos.cont=STRING            container uuid
+
+        # Flags
+        #  --daos.destroy                Destroy Container
+
+        # Optional arguments
+        #  --daos.group=STRING           server group
+        #  --daos.chunk_size=1048576     chunk size
+        #  --daos.oclass=STRING          object class
+
+        self.daos_pool_uuid = FormattedParameter("--daos.pool {}")
+        self.daos_svcl = FormattedParameter("--daos.svcl {}")
+        self.daos_cont = FormattedParameter("--daos.cont {}")
+        self.daos_group = FormattedParameter("--daos.group {}")
+        self.daos_chunk_size = FormattedParameter(" --daos.chunk_size {}")
+        self.daos_oclass = FormattedParameter("--daos.oclass {}")
+
+        # Module DFS
```

Comment:
Is there ever a case where a corresponding Module DAOS and Module DFS parameter would have different values?